### PR TITLE
Updated deployment instructions in readme

### PR DESCRIPTION
**Pull request by jamesefhawkins**
and made a few minor tweaks to functionality to highlight data warehouses

## Changes

*Please describe.*  
*If this affects the frontend, include screenshots.*  

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
- [ ] New/changed UI is decent on smartphones (viewport width around 360px)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,17 +5,17 @@
...
 
 <img align="left" src="https://user-images.githubusercontent.com/65415371/124739629-f43cde80-df11-11eb-9033-c5d1d7194f03.png" width="50px" />
 
 ## Quick Start
 
-1-click Heroku deploy:
+<a href="https://posthog.com/docs/self-host/overview#deploy">Deploy PostHog Self-Hosted</a>
 
-<a href="https://heroku.com/deploy?template=https://github.com/posthog/posthog"><img src="https://www.herokucdn.com/deploy/button.svg" width="250px" /></a>
+... or use [PostHog Cloud](https://app.posthog.com/signup) if you don't want to self-host.
```
Comment (tiina303) [inline README.md:18]:
```suggestion
... or use [PostHog Cloud](https://app.posthog.com/signup) if you don't need to self-host.
```

File: README.md [hunk 2]
```diff
@@ -76,42 +73,15 @@ This means you can know who is using your app, how they're using it, and where y
...
-* [Render](https://posthog.com/docs/deployment/deploy-render)
-
-<br /><br />
-
-<img align="left" src="https://user-images.githubusercontent.com/65415371/124740742-f94e5d80-df12-11eb-8ca9-7d2ad4f56e3d.png" width="50px" />
-
-## Production Deployment
-
-[See our Docs for production deployment options.](https://posthog.com/docs/deployment)
+You can [sign up for a free account](https://app.posthog.com/signup) if you don't want to host PostHog yourself.
```
Comment (tiina303) [inline README.md:84]: Shouldn't we still have a self hosting section here too?
```suggestion
You can [sign up for a free account](https://app.posthog.com/signup) if you don't want to host PostHog yourself.

<br /><br />

## Self-hosting

[See our Docs for self-hosted production deployment options.](https://posthog.com/docs/deployment)
```

**Review by tiina303 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (kpthatsme) [reply README.md]: agreed - was weird to callout cloud so specifically down here, changed this section to re-iterate the points from "Quick start"

**Comment by kpthatsme** [comment]
@jamesefhawkins I ran through this and tweaked a few things + resolved merge conflicts. I'm thinking we should just merge this and continue iterating from here.

**Comment by jamesefhawkins** [comment]
Just a quick thank you @kpthatsme for this

**Pull request merged**